veps/NNNN-vep-template/vep.md

```diff
@@ -92,10 +92,21 @@ For example:
 03-04-1922: Added support for doing even greater stuff. PR: <LINK>.
 -->
 
-## Feature lifecycle Phases
+## Graduation Requirements
 
 <!--
-How and when will the feature progress through the Alpha, Beta and GA lifecycle phases
+The requirements for graduating to each stage.
+Example:
+### Alpha
+- [ ] Feature gate guards all code changes
+- [ ] Initial implementation supporting only X and Y use-cases
+
+### Beta
+- [ ] Feature gate enabled by default
```

Comment:
As much as I want this to be the policy it isn't at the moment :)

Reply:
This is surprising and demands a different discussion.
I share your wish. :)

Reply:
Anyways, I've removed it from this PR. PTAL